scripts/cliupdate.php

```diff
@@ -30,139 +30,35 @@
  * ---------------------------------------------------------------------
  */
 
-if (in_array('--help', $_SERVER['argv'])) {
-   die("usage: ".$_SERVER['argv'][0]."  [ --force ] [ --lang=xx_XX ] [ --config-dir=/path/relative/to/script ] [--dev]\n");
+echo('Usage of this script has been deprecated, please use ../console.php glpi:database:update.' . PHP_EOL);
```

Comment:
Hum... Most of the time, cliupdate is called from glpi root directory; so `../` may be confusing

Reply:
As we see together, the old cliupdate script has been removed.